docker/Dockerfile.kubectl

```diff
@@ -2,8 +2,8 @@
 # Version is the base image version from the TLD Makefile
 ARG BASE_VERSION=latest
 
-# The following section is used as base image if BASE_DISTRIBUTION=default
-FROM docker.io/istio/base:${BASE_VERSION} as default
```

Comment:
It seems the `default` or `ubuntu` is not used in this file.

Reply:
Thats true, I think its just added for consistency. Adding `as ubuntu` and not referencing it doesn't have any downside as far as I know. The problem is the kubectl image hasn't been migrated to distroless yet (not because you can't have kubectl in distroless, but when we use kubectl we depend on shell stuff)